--- g2clib-1.2.0/makefile	2009-06-10 06:27:51.000000000 +1200
+++ g2clib-1.2.0-modified/makefile	2010-07-07 09:32:18.000000000 +1200
@@ -19,11 +19,12 @@
 #   appropriate compiler flag to allow the use of "//" comment indicators.
 #
 #CFLAGS=-q64 -O3 -qarch=auto -qcpluscmt $(INC) $(DEFS)
-CFLAGS= $(INC) $(DEFS)
+CFLAGS= $(INC) $(DEFS) -fPIC
 
 CC=cc
 LIB=libgrib2c.a
-ARFLAGS=-X64
+#ARFLAGS=-X32_64
+ARFLAGS=ruv
 
 all: $(LIB)
 
@@ -80,6 +81,6 @@
 
 .c.a:
 	$(CC) -c $(CFLAGS) $<
-	ar $(ARFLAGS) -ruv $@ $*.o
+	ar $(ARFLAGS) $@ $*.o
 	rm -f $*.o
 
